minimal-launchpad/minimal_ui_index.js

```diff
@@ -215,21 +215,23 @@ async function connectToDevice() {
     spinner.style.alignItems = "center";
 
     try {
+        const commonLoaderOptions = {
+            transport: transport,
+            baudrate: 460800,
+            terminal: espLoaderTerminal,
+        };
+
         let loaderOptions;
+
         if (config.portConnectionOptions?.length) {
```

Comment:
Do we require this if - else block now? in both cases it has commonLoaderOptions

Reply:
I think as for expresslink, there is the special requirement of serialOptions need to be passed.